common/transformations/orientation.py

```diff
@@ -1,6 +1,7 @@
 # pylint: skip-file
 import numpy as np
 
+from typing import Callable
```

Comment:
our convention here is std + third party libs then openpilot imports
```suggestion
import numpy as np
from typing import Callable

```